docs/_static/hasura-custom.css

```diff
@@ -22,7 +22,7 @@ h1, h2, h3, h4, h5, h6 {
 body, p, #docs-content, #sidebar, #sidebar a,
```

Comment:
why do we have global css changes going in with this?

Reply:
I'll move to a separate PR